src/services/base/base.service.js

```diff
@@ -845,6 +898,16 @@ module.exports.merge = async (args, {req}, resource_name, collection_name) => {
                     }
                 ]
             };
+            const currentDate = moment.utc().toDate().format('YYYY-MM-DD');
+            await sendToS3(resource_name,
```

Comment:
Two blocking `await` calls you want to wrap to run concurrently. e.g.,

`Promise.all([sendToS3(firstCall), sentToS3(secondCall)])`